app/middleware/setLocals.js

```diff
@@ -3,6 +3,8 @@ const backLinkUtils = require('../lib/utils/backLink');
 function fromRequest(req, res, next) {
   res.locals.search = req.query.search;
   res.locals.postcodeSearch = req.query.postcode;
+  res.locals.resultsReturnedCount = 30;
```

Comment:
? rename to resultsLimit.

Also, given that this isn't _fromRequest_ is would be better to have it in `config/config.js` and `app/middleware/locals.js` as for the analytics Id's?

If pulled from an env var it will need a default (e.g. `resultsLimit: process.env.RESULTS_LIMIT || 30`).

Same probably applies to the other hard coded values but they can have an issue raised.